win32/GNUmakefile

```diff
@@ -1420,23 +1420,27 @@ else
 	echo #define NVgf "g"&& \
 	echo #undef I_QUADMATH&& \
 	echo #undef USE_QUADMATH&& \
-	echo #undef USE_LONG_DOUBLE)>> config.h
+	echo #undef USE_LONG_DOUBLE)>> $(MINIDIR)\config.h
 endif
 ifeq ($(USE_CPLUSPLUS),define)
 	@(echo #define USE_CPLUSPLUS&& \
-	echo #endif)>> config.h
+	echo #endif)>> $(MINIDIR)\config.h
 else
 	@(echo #undef USE_CPLUSPLUS&& \
-	echo #endif)>> config.h
+	echo #endif)>> $(MINIDIR)\config.h
 endif
 #separate line since this is sentinal that this target is done
 	@rem. > $(MINIDIR)\.exists
 
+$(MINIDIR)\\%$(o): %.c
```

Comment:
is the double backslash required here? just above it doesnt seem to be required.  Would a comment be in order?

Anyway, other than that LGTM.

Reply:
A single `\` quotes the `%`, so an extra `\` is needed to quote the `\`.

https://www.gnu.org/software/make/manual/html_node/Selective-Search.html#index-_005c-_0028backslash_0029_002c-to-quote-_0025